rfcs/QueryLevelNullability.md

```diff
@@ -1,100 +1,158 @@
 # RFC: Query Level Nullability 
 
-**Proposed by:** 
+**Proposed by:**
+
+- [Alex Reilly](<social or github link here>) - Yelp iOS
 - [Liz Jakubowski](https://github.com/lizjakubowski) - Yelp iOS
-- [Sanae Rosen](<social or github link here>) - Yelp Android
 - [Mark Larah](https://github.com/magicmark) - Yelp Web
-- [Alex Reilly](<social or github link here>) - Yelp iOS
+- [Sanae Rosen](<social or github link here>) - Yelp Android
 - [Wei Xue](<social or github link here>) - Yelp iOS
 
-This RFC proposes a syntactical construct for GraphQL clients to express the **nullability** of schema fields requested in a query.
+This RFC proposes a syntactical construct for GraphQL clients to express the **nullability** of schema fields requested
+in a query.
```

Comment:
a lot of changes here are just newlines (I used 120 characters since it seemed to be what you were using too)

Reply:
I was honestly doing all of my editing in GH, so I didn't have a hard line limit. Thanks for cleaning up!